modules/xmpp/SDP.spec.js

```diff
@@ -239,7 +241,9 @@ a=ssrc:3758540092 mslabel:mixedmslabel
             const offer = createStanzaElement(stanza);
             const sdp = new SDP('');
 
-            sdp.fromJingle($(offer).find('>jingle'));
+            console.log('#####################', offer.querySelector);
```

Comment:
debug leftover?

Reply:
yeah, was debugging why querySelector wasnt selecting the xmlns